examples/5-nextjs/src/models/RootStore.base.ts

```diff
@@ -17,12 +17,29 @@ type Refs = {
   users: ObservableMap<string, UserModelType>
 }
 
+
+/**
+* Enums for the names of base graphql actions
+*/
+export enum RootStoreBaseQueries {
+queryTodos="queryTodos",
+queryDoneTodos="queryDoneTodos",
+queryUser="queryUser",
+queryUsers="queryUsers"
+}
+export enum RootStoreBaseMutations {
+mutateTodos="mutateTodos",
+mutateDoneTodos="mutateDoneTodos",
+mutateUser="mutateUser",
+mutateUsers="mutateUsers"
+}
+
 /**
 * Store, managing, among others, all the objects received through graphQL
 */
 export const RootStoreBase = withTypedRefs<Refs>()(MSTGQLStore
   .named("RootStore")
-  .extend(configureStoreMixin([['Todo', () => TodoModel], ['User', () => UserModel]], ['Todo', 'User']))
+  .extend(configureStoreMixin([['Todo', () => TodoModel], ['User', () => UserModel]], ['Todo', 'User'], "js"))
```

Comment:
Ditto for this change. I don't think this was updated the last time generation code was updated